src/core/public/styles/rendering/_base.scss

```diff
@@ -33,6 +33,8 @@
   flex-grow: 1;
   z-index: 0; // This effectively puts every high z-index inside the scope of this wrapper to it doesn't interfere with the header and/or overlay mask
   position: relative; // This is temporary for apps that relied on this being present on `.application`
+
+  --euiFixedHeadersOffset: calc(var(--kbnHeaderOffset) + var(--kbnProjectHeaderAppActionMenuHeight));
```

Comment:
For everything inside the .kbnAppWrapper we add the headerActionMenu height to the offset

Reply:
Rather than overriding the existing CSS variable, I'd rather create a new one, e.g. `kbnAllHeadersOffset` or similar. My concern here is that various EUI internals use the `--euiFixedHeadersOffset` var (e.g. flyouts, full screen modes, etc) and most of these elements **should** sit on top of the app toolbar and any sticky menu items, not below.